src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerArgumentCollection.java

```diff
@@ -123,4 +124,15 @@ public ReadThreadingAssembler createReadThreadingAssembler() {
     @Argument(fullName = MAX_MNP_DISTANCE_LONG_NAME, shortName = MAX_MNP_DISTANCE_SHORT_NAME,
             doc = "Two or more phased substitutions separated by this distance or less are merged into MNPs.", optional = true)
     public int maxMnpDistance = getDefaultMaxMnpDistance();
+
+    @Advanced
+    @Argument(fullName = ALLELE_EXTENSION_LONG_NAME,
+            doc = "Likelihood and read-based annotations will only take in consideration reads " +
+                    "that overlap the variant or any base on further than this distance expressed in base pairs",
+            optional = true)
+    public int informativeReadOverlapRadius = 0;
+
+    @Advanced
+    @Argument(fullName= USE_FILTERED_READS_FOR_ANNOTATIONS_LONG_NAME, doc = "Use the contamination-filtered read maps for the purposes of annotating variants", optional=true)
```

Comment:
We deliberately took the option to do this out of Mutect2 some time ago.  Unless there's a very good reason I would prefer to keep it in `HaplotypeCallerArgumentCollection`.

Reply:
done.